scss/_cards.scss

```diff
@@ -42,6 +44,19 @@
   }
 }
 
+.card-warning, .card-default {
+  .card-blockquote {
+    color: $card-blockquote-color;
+  }
+}
+
+.card-block {
+  > .card-block {
```

Comment:
What's the use-case for this, `card-block`s shouldn't be nested like this. If the idea is to get padding on the left and right, `px-*` can be used.